tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/test_lockscreen_unlock_to_camera.py

```diff
@@ -25,6 +26,7 @@ def test_unlock_to_camera(self):
         lock_screen = LockScreen(self.marionette)
         lock_screen.switch_to_frame()
         camera = lock_screen.unlock_to_camera()
+        Wait(self.marionette).until(lambda m: self.device.is_locked == False)
```

Comment:
Now, the problem with doing above is that for test_lockscreen_unlock_to_emergency_call_screen.py.  It is passcode locked, but instead of typing passcode to unlock to homescreen, the user makes an emergency call.  For these reasons, I think it is better to keep the methods in app.py atomic, and check for the unlock status at the script level.  What do you think?